Userland/Libraries/LibWeb/Loader/LoadRequest.h

```diff
@@ -37,6 +38,9 @@ class LoadRequest {
     void start_timer() { m_load_timer.start(); };
     Time load_time() const { return m_load_timer.elapsed_time(); }
 
+    Optional<Page&> page() const { return m_page; };
```

Comment:
`Optional<Page&> const&`?

Reply:
It can't be a `const&`, the `value()` method of a `const optionnal` return a constant `Page` and a mutable one is required down the road